dbft.go

```diff
@@ -74,15 +76,19 @@ func (d *DBFT[H, A]) addTransaction(tx block.Transaction[H]) {
 	}
 }
 
-// Start initializes dBFT instance and starts protocol if node is primary. It
-// accepts a timestamp of the previous block.
+// Start initializes dBFT instance and starts the protocol if node is primary.
+// It accepts the timestamp of the previous block. It should be called once
+// per DBFT lifetime.
 func (d *DBFT[H, A]) Start(ts uint64) {
 	d.cache = newCache[H, A]()
 	d.InitializeConsensus(0, ts)
 	d.start()
 }
 
-// InitializeConsensus initializes dBFT instance.
+// InitializeConsensus reinitializes dBFT instance and the given view with the
+// given timestamp of the previous block. It's used if the current consensus
+// state is outdated (which can happen when a new block is received by node
+// before completing its assembly via dBFT). View 0 is expected to be used.
```

Comment:
> (which can happen when a new block is received by node before completing its assembly via dBFT)

Actually, it should be called even if dBFT completes its assembly before receiving this block via network. dBFT can't increase its height by himself. Maybe it should be added to the comment.